tests/rules/javascript/test.no_eval.js

```diff
@@ -5,6 +5,9 @@ import { DANGEROUS_EVAL } from 'messages';
 // These rules were mostly copied and adapted from
 // https://github.com/eslint/eslint/blob/master/tests/lib/rules/no-eval.js
 // Please make sure to keep them up-to-date and report upstream errors.
+
+// ??? should I renew validCodes and invalidCodes as
+// they are now in the source? There is some difference.
```

Comment:
Should I renew validCodes and invalidCodes as they are now in the source? There is some difference.